src/content/docs/en/reference/api-reference.mdx

```diff
@@ -52,14 +52,19 @@ Markdown files have the following interface:
 export interface MarkdownInstance<T extends Record<string, any>> {
   /* Any data specified in this file's YAML frontmatter */
 	frontmatter: T;
-  /* The file path of this file */
+  /* The absolute file path of this file */
 	file: string;
   /* The rendered path of this file */
 	url: string | undefined;
   /* Astro Component that renders the contents of this file */
 	Content: AstroComponent;
```

Comment:
I hesitated to update this type (I believe it is present elsewhere as well). `AstroComponent` does not exist, the real type is `AstroComponentFactory`. Should I update all the occurrences or has the type been simplified for documentation purposes?

Reply:
I think using the accurate name is not a bad idea. Admittedly `AstroComponentFactory` is not a type we really document, so either way it doesn’t matter **_too_** much, but might as well use the same name we use in the code base in case anyone does want to search for it.